CHANGELOG.md

```diff
@@ -1,3 +1,6 @@
+# React Native Kommunicate Chat v2.5.7
+- [Android] Fixed Assignee names are not updating if clientConversationId is set in buildConversation.
```

Comment:
Fixed assignee names do not update if clientConversationId is set in buildConversation.